server-common/src/main/java/org/apache/kafka/server/common/GroupVersion.java

```diff
@@ -29,6 +29,8 @@ public enum GroupVersion implements FeatureVersion {
 
     public static final String FEATURE_NAME = "group.version";
 
+    public static final GroupVersion LATEST_PRODUCTION = GV_1;
```

Comment:
Currently, latest production for GV is GV_0. So, we actually want to move it to GV_1?

Reply:
This is not correct. The latest production for GV is 1. 
https://github.com/apache/kafka/blob/6a43a6bddfa04a15832ccccf8742919d1666fd86/server-common/src/main/java/org/apache/kafka/server/common/GroupVersion.java#L28
https://github.com/apache/kafka/blob/6a43a6bddfa04a15832ccccf8742919d1666fd86/server-common/src/main/java/org/apache/kafka/server/common/MetadataVersion.java#L253